gc/base/GCExtensionsBase.hpp

```diff
@@ -355,6 +355,11 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 
 	const char* gcModeString;
 	uintptr_t splitFreeListSplitAmount;
+	/*
+	 * the number of split freelist, set by command line option, or determined heuristically based on the number of cpus,
```

Comment:
class field comments have a specific format. they are appended to the definition, they are in line start with '/**<'

Reply:
Does that mean it can exceed the maximum number of characters per line?

Reply:
For this field this is enough /**< The number of split freelists in heap (or in tenure for gencon) */
For the Forced field: /**< Flag to distinguish if splitFreeListAmount is externally enforced (for example, specified by command line) or determined heuristically */

No space here to go into heuristic details.